CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ None
 
 ### Added
 
+- Added callback-methods `paper_trail_update` `paper_trail_create` `paper_trail_destroy`
+  instead of has_paper_trail
+  [#593](https://github.com/airblade/paper_trail/pull/607)
```

Comment:
- Should read `paper_trail_on_update`.
- PR number in link should be 614, right?